index.html

```diff
@@ -35,7 +35,7 @@
       specStatus: "ED",
       // We can only indirectly test conformance, so we rely on certain tests
       // in the Payment Request API.
-      testSuiteURI: "https://w3c-test.org/payment-request/",
+      testSuiteURI: "https://w3c-test.org/payment-method-id/",
```

Comment:
see my comment in the related bug about adding the implementation report property too.